### ISSUE TITLE
Crash due to old `AppCompat` Library

### ISSUE THREAD

**Issue by Iamlooker** [zhanghai/AndroidFastScroll#49]
## Expected Behavior
Should run flawlessly and show a fast scroller.

## Actual Behavior
Crash on open the screen with the fast scroller.

## Steps to Reproduce the Problem

1. Add this library to any project with `appcompat:1.6.0-rc01`
2. Run the app

## Specifications

- Version: 1.1.8
- Platform: Any version of Android

I am just guessing that the issue is with AppCompat versioning because I encountered this issue after updating to latest library version

```
java.lang.NoClassDefFoundError: Failed resolution of: Landroidx/appcompat/graphics/drawable/DrawableWrapper;
    at me.zhanghai.android.fastscroll.FastScrollerBuilder.useDefaultStyle(FastScrollerBuilder.java:123)
    at me.zhanghai.android.fastscroll.FastScrollerBuilder.<init>(FastScrollerBuilder.java:61)
```

**Comment by zhanghai** [comment]
That's because of [aosp/2120177](https://android-review.googlesource.com/c/platform/frameworks/support/+/2120177). For now you can get a ToT build of this library with Jitpack, while I'll release a new version when AppCompat 1.6.0 gets a stable release.

**Comment by zhanghai** [comment]
Fixed in 1.1.9.
